spec/fixtures/module/preload-eventemitter.js

```diff
@@ -0,0 +1,11 @@
+(function () {
+  const { EventEmitter } = require('events');
+  const emitter = new EventEmitter();
+  let rendererEventEmitterProperties = '';
+  let currentObj = emitter;
+  do {
+    Object.getOwnPropertyNames(currentObj).map(property => { rendererEventEmitterProperties += property + ';'; });
+  } while ((currentObj = Object.getPrototypeOf(currentObj)));
+  const { ipcRenderer } = require('electron');
+  ipcRenderer.send('answer', rendererEventEmitterProperties);
```

Comment:
This can be an array of strings, no need to flatten it out to a string.

```suggestion
  let rendererEventEmitterProperties = [];
  let currentObj = emitter;
  do {
    rendererEventEmitterProperties.push(...Object.getOwnPropertyNames(currentObj));
  } while ((currentObj = Object.getPrototypeOf(currentObj)));
  const { ipcRenderer } = require('electron');
  ipcRenderer.send('answer', rendererEventEmitterProperties);
```